pkg/descheduler/evictions/evictions.go

```diff
@@ -77,29 +81,34 @@ func NewPodEvictor(
 
 // NodeEvicted gives a number of pods evicted for node
 func (pe *PodEvictor) NodeEvicted(node *v1.Node) uint {
-	return pe.nodepodCount[node.Name]
+	return pe.nodePodCount[node.Name]
 }
 
 // TotalEvicted gives a number of pods evicted through all nodes
 func (pe *PodEvictor) TotalEvicted() uint {
-	var total uint
-	for _, count := range pe.nodepodCount {
-		total += count
-	}
-	return total
+	return pe.totalPodCount
```

Comment:
I want to confirm one thing. The change in this PR maintains the totalPodCount field. Can we ensure that the totalPodCount field is consistent with the result obtained by the previous method?

Reply:
`pod.Spec.Node == ""` should we considered this situation?
When this happens, the two maintained fields (nodePodCount totalPodCount) will be inconsistent
```go
         if pod.Spec.NodeName != "" {
		
		pe.nodePodCount[pod.Spec.NodeName]++
	}
	pe.namespacePodCount[pod.Namespace]++
	pe.totalPodCount++

```

Reply:
> Can we ensure that the totalPodCount field is consistent with the result obtained by the previous method?

yes

Reply:
> When this happens, the two maintained fields (nodePodCount totalPodCount) will be inconsistent

I think this is reasonable.  If `if pod.Spec.NodeName != "" {` occurs, `totalPodCount in TotalEvicted()`  is a more accurate value.

Reply:
Understood, thanks